keras_nlp/models/bert/bert_backbone_test.py

```diff
@@ -52,7 +52,7 @@ def setUp(self):
     def test_valid_call_bert(self):
         self.model(self.input_batch)
         # Check default name passed through
-        self.assertEqual(self.model.name, "backbone")
```

Comment:
Maybe `assertRegexMatch` here? If we end up creating two bert models above, this could have a name like `"backbone_x"`. Regex match seems more robust.